src/Build.OM.UnitTests/Construction/SolutionFile_Tests.cs

```diff
@@ -91,6 +78,13 @@ public void InvalidSolutionFilters(string slnfValue, string exceptionReason)
                 TransientTestFolder folder = testEnvironment.CreateFolder(createFolder: true);
                 TransientTestFile sln = testEnvironment.CreateFile(folder, "Dev.sln");
                 TransientTestFile slnf = testEnvironment.CreateFile(folder, "Dev.slnf", slnfValue.Replace(@"C:\\notAPath\\MSBuild.Dev.sln", sln.Path.Replace("\\", "\\\\")));
+                try
+                {
+                    SolutionFile.Parse(slnf.Path);
+                } catch (Exception ex)
+                {
+                    Console.WriteLine(ex);
+                }
```

Comment:
??

Reply:
I need to remember to take out code only used for testing _before_ committing.